config/webpack/development.js

```diff
@@ -4,9 +4,9 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'development'
 
 const environment = require('./environment')
 
-// environment.config.devServer.https = {
-//   key: fs.readFileSync('./config/localhost/https/localhost.key'),
-//   cert: fs.readFileSync('./config/localhost/https/localhost.crt')
-// }
+environment.config.devServer.https = {
+  key: fs.readFileSync('./config/localhost/https/localhost.key'),
```

Comment:
Just wondering can those string value be passed from yml, or have the yml comment to state its overridden here

Reply:
It doesn't work:

```
/Users/tvararu/dfe/manage-courses-frontend/node_modules/webpack-dev-server/bin/webpack-dev-server.js:403
    throw e;
    ^

Error: error:0906D06C:PEM routines:PEM_read_bio:no start line
    at Object.createSecureContext (_tls_common.js:113:17)
    at Server (_tls_wrap.js:870:27)
    at Server (https.js:62:14)
    at Server._init (/Users/tvararu/dfe/manage-courses-frontend/node_modules/spdy/lib/spdy/server.js:63:10)
    at new Server (/Users/tvararu/dfe/manage-courses-frontend/node_modules/spdy/lib/spdy/server.js:26:10)
    at Function.create (/Users/tvararu/dfe/manage-courses-frontend/node_modules/spdy/lib/spdy/server.js:31:12)
    at Object.create [as createServer] (/Users/tvararu/dfe/manage-courses-frontend/node_modules/spdy/lib/spdy/server.js:287:27)
    at new Server (/Users/tvararu/dfe/manage-courses-frontend/node_modules/webpack-dev-server/lib/Server.js:484:30)
    at startDevServer (/Users/tvararu/dfe/manage-courses-frontend/node_modules/webpack-dev-server/bin/webpack-dev-server.js:396:14)
    at processOptions (/Users/tvararu/dfe/manage-courses-frontend/node_modules/webpack-dev-server/bin/webpack-dev-server.js:360:5)
```

Reply:
```
environment.config.devServer.https = {

 key: fs.readFileSync(environment.config.devServer.https.custom.key),

```

sorry i meant something like that...

Reply:
Ah, hmm... I'll give it a go, but first I would like it to just work. That gave me a last resort idea to maybe fix it. 💡

Reply:
Right, looked at this for a bit but it's buggy and weird, just leaving it as is for now.